core/kernel.rbs

```diff
@@ -488,7 +490,8 @@ module Kernel : BasicObject
   #     Hash(nil)              # => {}
   #     Hash([])               # => {}
   #
-  def self?.Hash: [K, V] (Object x) -> ::Hash[K, V]
+  def self?.Hash: [K, V] (nil | [] _empty) -> Hash[K, V]
```

Comment:
`nil` and `[]` are already `_ToHash[untyped, untyped]`s. But if you want to add them here explicitly, note the `[K, V]` are unused.

```suggestion
  def self?.Hash: (nil | [] _empty) -> Hash[untyped, untyped]
```

Reply:
Oh right they aren’t `_ToHash`es, just `_ToH`s.
So this method only checks for `#to_hash` but not `#to_h`, unlike `Integer` and `Array`?